.github/workflows/perf-test-eks.yaml

```diff
@@ -31,7 +31,7 @@ jobs:
       - name: make dev/tools
         run: make dev/tools
       - name: Start cluster
-        run: ENV=eks make start-cluster
+        run: ENV=eks TF_VARS="aws_role_id=${{ secrets.AWS_ACCOUNT_ID }}" make start-cluster
```

Comment:
How is the account id a role id?

Reply:
secret naming is actually wrong, it holds ROLE id atm, changed